data-prepper-plugins/cloudwatch-logs/src/test/java/org/opensearch/dataprepper/plugins/sink/configuration/CwlSinkConfigTest.java

```diff
@@ -0,0 +1,38 @@
+package org.opensearch.dataprepper.plugins.sink.configuration;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.opensearch.dataprepper.plugins.sink.config.CwlSinkConfig;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+public class CwlSinkConfigTest {
+    public static final String LOG_GROUP = "testGroup";
+    public static final String LOG_STREAM = "testStream";
+    public static final String BUFFER_TYPE = "in_memory";
+    public static final int BATCH_SIZE = 10;
+    public static final int MAX_RETRIES = 10;
```

Comment:
How are these values used ?

Reply:
Sorry, it seems that while working on the testing, I had left these values here although they were never used. I will get rid of these for the next revision.